public/locales/en/panel-flightcontrol.json

```
@@ -0,0 +1,25 @@
+{
+  "toggle-flight-label": "Toggle flight control",
+  "description": {
+    "intro": "Interact with the highlighted area to control the camera.",
+    "mouse-title": "Mouse controls:",
+    "mouse-body": {
+      "text": "Click and drag to rotate. Hold",
+      "keybinds": ["Shift to zoom (y-axis) or roll (x-axis)", "Ctrl to pan"]
```

I think we should try to always linebreak arrays, to make life easier for the person editing the file next time
```suggestion
      "keybinds": [
        "Shift to zoom (y-axis) or roll (x-axis)",
        "Ctrl to pan"
      ]
```

---

Ah ok, ignoer then